app/Lib/Rscripts/analyse.R

```diff
@@ -94,9 +136,14 @@ Analyse <- function(key, input.answers, number.answeroptions) {
       percentage.answer.options <- 0
       corrected.item.tot.cor.answ.option <- 0
     }
-    
-    # Computes Cronbach's Alpha
-    cronbach <- round(cronbach(input.correct)$alpha, digits = 3)
+
+    # Computes Cronbach's Alpha for overall test
+    # If there is no variance in the total score returns -100
+    if(!all(rowSums(input.correct) == rowSums(input.correct)[1])){
+    	cronbach <- round(cronbach(input.correct)$alpha, digits = 3)
+    } else {
+    	cronbach <- -100
```

Comment:
Changed to -9. The reason for -100 was that it should actually be -Inf, Changed it to the smallest possible value.